README.md

```diff
@@ -143,10 +156,14 @@ Optional parameters (not required for display):
 * *`reveal-when-loaded`*: When *`poster`* and *`preload`* are specified, hide
   the poster and show the model once the model has been loaded.  See [On
   Loading](#on-loading) for more information.
+* *`unstable-webxr`*: Enables the ability to view the model in AR via the experimental
+  [WebXR Device API], currently [implemented only in Chrome Canary](https://developers.google.com/web/updates/2018/06/ar-for-the-web).
+  See [Augmented Reality](#augmented-reality).
 
 All attributes have a corresponding property in camel-case format. For example,
 the `background-color` attribute can also be configured using the
-`backgroundColor` property.
+`backgroundColor` property, including acronyms in names (e.g. `unstable-webxr`
```

Comment:
"including acronyms in names" seems potentially confusing, at least to me, and I'm not sure that we're applying that rule consistently (see ar and ios-src). Should we clarify here or put the property name next to each attribute?

Reply:
Good point -- `ar` is gone, but `unstableWebXR` and `iosSrc` are inconsistent -- I can change `unstableWebxr`, but should clarify these instances since I don't know if these are technically camel case conversions @cdata

Reply:
This topic is super bike-sheddy. `unstableWebXr` is the indisputably correct color.

Reply:
Super bike-sheddy is a good way to put it (although `unstableWebXr` is my preferred color as well, fwiw).

That being said, I'm happy with anything as long as it's clear to developers.